src/InboundServer/handlers.js

```diff
@@ -55,6 +63,11 @@ const getParticipantsByTypeAndId = async (ctx) => {
  */
 const getPartiesByTypeAndId = async (ctx) => {
     // kick off an asyncronous operation to handle the request
+    const histTimerEnd = Metrics.getHistogram(
+        'inbound_get_parties_type_id',
```

Comment:
Any reason that you are not using the information that you set in the Route Map instead....it seems that this is using the same metric names for the event framework?

Reply:
decided not to go with this option as name in route map requires the path to access